Library/Homebrew/cmd/update.rb

```diff
@@ -453,8 +453,8 @@ def dump_formula_report(key, title)
       end
 
       # Append suffix '(installed)' to list items of installed formulae.
-      formula_installed_index.each do |index|
-        formula[index] += " (installed)"
+      formula_installed_index.map! do |index|
+        pretty_installed(formula[index])
       end
```

Comment:
I'm not sure which thought went into this, but it doesn't make sense to me to transform the indices of the installed formula into the formatted text and then to use this text as indices to access the `formula` array as is done in the following code line. The thing that would make the most sense to me would be to have something like:

```ruby
formula_installed_index.each do |index|
  formula[index] = pretty_installed(formula[index])
end
```

But even then, the code below would require adjustment and `puts_columns` wouldn't be able to create a proper multi-column layout due to the preformatted list items (like explained in my other comment).